src/style/_fonts.scss

```diff
@@ -0,0 +1,32 @@
+@font-face{
```

Comment:
I'm being pedantic here... but I imagine the OS guys will be as well.
Should make sure your white-spacing is consistent i.e "1 white-space line between each font-face declaration"
...also swagger-ui seems to have a standard of 4 space indentation in sass files.